app/Console/Commands/Node/MakeNodeCommand.php

```diff
@@ -0,0 +1,102 @@
+     */
+    protected $signature = 'p:node:make
+                            {--name= : A name to identify the node.}
+                            {--description= : A description to identify the node.}
+                            {--locationId= : A valid locationId.}
+                            {--fqdn= : The domain name (e.g node.example.com) to be used for connecting to the daemon. An IP address may only be used if you are not using SSL for this node.}
+                            {--public= : Should the node be public or private? (public=1 / private=0).}
+                            {--scheme= : Which scheme should be used? (Enable SSL=https / Disable SSL=http).}
+                            {--proxy= : Is the daemon behind the cloudflare proxy? (Yes=1 / No=0).}
+                            {--maintenance= : Should maintenance mode be enabled? (Enable Maintenance mode=1 / Disable Maintenance mode=0).}
+                            {--maxMemory= : Set the max memory amount.}
+                            {--overallocateMemory= : Enter the amount of ram to overallocate (% or -1 to overallocate the maximum).}
+                            {--maxDisk= : Set the max disk amount.}
+                            {--overallocateDisk= : Enter the amount of disk to overallocate (% or -1 to overallocate the maximum).}
+                            {--uploadSize= : Enter the maximum upload filesize.}
+                            {--daemonListeningPort= : Enter the wings listening port.}
+                            {--daemonSFTPPort= : Enter the wings SFTP listening port.}
+                            {--daemonBase= : Enter the base folder.}';
+
+    /**
+     * @var string
+     */
+    protected $description = 'Creates a new node on the system via the CLI.';
+
+    /**
+     * Create a new command instance.
+     */
+    public function __construct(NodeCreationService $creationService)
+    {
+        parent::__construct();
+
+        $this->creationService = $creationService;
+    }
+
+    /**
+     * Handle the command execution process.
+     *
+     * @throws \Pterodactyl\Exceptions\Model\DataValidationException
+     */
+    public function handle()
+    {
+        $name = $this->option('name') ?? $this->ask(trans('command/messages.node.ask_name'));
+        $description = $this->option('description') ?? $this->ask(trans('command/messages.node.ask_description'));
+        $location_id = $this->option('locationId') ?? $this->ask(trans('command/messages.node.ask_location_id'));
+        $fqdn = $this->option('fqdn') ?? $this->ask(trans('command/messages.node.ask_fqdn'));
+        if (!filter_var(gethostbyname($fqdn), FILTER_VALIDATE_IP)) {
+            $this->error(trans('admin/node.validation.fqdn_not_resolvable'));
+            return;
+        }
+        $public = $this->option('public') ?? $this->ask(trans('command/messages.node.ask_public'));
+        $scheme = $this->option('scheme') ?? $this->ask(trans('command/messages.node.ask_scheme'));
```

Comment:
This can use the pre-defined selectors rather than just being a generic input. Should auto-suggest `http(s)`